Add java helper as for nodejs

gcollin opened the pull request:
Seems like some other PR removed it (my fault)

Commit:
Add java helper as for nodejs

yalh76 commented:
Can you rename it to ynh_install_java to be more consistent to others language install

yalh76 commented on ynh_java/ynh_java:
```diff
@@ -0,0 +1,128 @@
+    # Allow alias to be used into bash script
+    shopt -s expand_aliases
+
+    # Create an alias for the specific version of node and a variable as fallback
```
there is some references of node instead of java ...

yalh76 replied:
fixed

yalh76 commented:
ynh_remove_java is missing

yalh76 commented:
you should better base this helper from ynh_install_go of ynh_install_ruby

gcollin commented:
Ok, I'll check with install_go and updates

Commit:
Added remove java

gcollin commented:
@yalh76 I renamed the scripts and added ynh_remove_java as well. However, I don't need to clean-up versions like for go because Java is installed using the yunohost package dependencies. It's removed when no other apps are using it.

yalh76 committed:
Typo

yalh76 reviewed (approved):
Great work, thanks a lot :)
